db/re/skill_db.yml

```
@@ -41846,6 +41846,725 @@ Body:
+        - Level: 3
+          Amount: 60
+        - Level: 4
+          Amount: 65
+        - Level: 5
+          Amount: 70
+      ItemCost:
+        - Item: Soa_Charm
+          Amount: 1
+    Status: Talisman_of_Warrior
+  - Id: 5420
+    Name: SOA_TALISMAN_OF_MAGICIAN
+    Description: Talisman Of Magician
+    MaxLevel: 5
+    TargetType: Support
+    DamageFlags:
+      NoDamage: true
+    Flags:
+      PartyOnly: true
+    Range: 9
+    Hit: Single
+    HitCount: 1
+    GiveAp: 2
+    CastCancel: true
+    CastTime: 1500
+    AfterCastActDelay: 500
+    Duration1:
+      - Level: 1
+        Time: 180000
+      - Level: 2
+        Time: 210000
+      - Level: 3
+        Time: 240000
+      - Level: 4
+        Time: 270000
+      - Level: 5
+        Time: 300000
+    FixedCastTime: 1000
+    Requires:
+      SpCost:
+        - Level: 1
+          Amount: 50
+        - Level: 2
+          Amount: 55
+        - Level: 3
+          Amount: 60
+        - Level: 4
+          Amount: 65
+        - Level: 5
+          Amount: 70
+      ItemCost:
+        - Item: Soa_Charm
+          Amount: 1
+    Status: Talisman_of_Magician
+  - Id: 5421
+    Name: SOA_SOUL_GATHERING
+    Description: Soul Gathering
+    MaxLevel: 5
+    TargetType: Self
+    DamageFlags:
+      NoDamage: true
+    Hit: Single
+    HitCount: 1
+    CastCancel: true
+    FixedCastTime: 1000
+    CastTime: 1500
+    Cooldown:
+      - Level: 1
+        Time: 5000
+      - Level: 2
+        Time: 4000
+      - Level: 3
+        Time: 3000
+      - Level: 4
+        Time: 2000
+      - Level: 5
+        Time: 1000
+    Requires:
+      SpCost: 
+        - Level: 1
+          Amount: 70
+        - Level: 2
+          Amount: 75
+        - Level: 3
+          Amount: 80
+        - Level: 4
+          Amount: 85
+        - Level: 5
+          Amount: 90
+  - Id: 5422
+    Name: SOA_TOTEM_OF_TUTELARY
+    Description: Totem Of Tutelary
+    MaxLevel: 5
+    Type: Magic
+    TargetType: Ground
+    DamageFlags:
+      NoDamage: true
+    Range: 11
+    Hit: Single
+    HitCount: 1
+    SplashArea: 2
+    ActiveInstance: 1
+    CastCancel: true
+    CastTime: 3500
+    Duration1:
+      - Level: 1
+        Time: 18000
+      - Level: 2
+        Time: 21000
+      - Level: 3
+        Time: 24000
+      - Level: 4
+        Time: 27000
+      - Level: 5
+        Time: 30000
+    Cooldown: 15000
+    FixedCastTime: 1000
+    Requires:
+      SpCost:
+        - Level: 1
+          Amount: 76
+        - Level: 2
+          Amount: 92
+        - Level: 3
+          Amount: 108
+        - Level: 4
+          Amount: 124
+        - Level: 5
+          Amount: 140
+      ItemCost:
+        - Item: Soa_Charm
+          Amount: 10
+    Unit:
+      Id: Totem_Of_Tutelary
+      Range:
+        - Level: 1
+          Size: 2
+        - Level: 2
+          Size: 2
+        - Level: 3
+          Size: 3
+        - Level: 4
+          Size: 3
+        - Level: 5
+          Size: 4
+      Interval: 1000
```

```suggestion
      Interval: 500
```

According to @datawulf